src/main/java/pl/edu/agh/firecell/engine/algorithm/FirePropagator.java

```diff
@@ -56,7 +58,7 @@ public int computeFirePillar(State oldState, Cell oldCell, Vector3i cellIndex, i
 
     public static Stream<Vector3i> getBurningHorizontalNeighbours(State oldState, Vector3i cellIndex){
         return Stream
-                .concat(NeighbourUtils.neighboursStream(cellIndex, NeighbourUtils.Axis.X), NeighbourUtils.neighboursStream(cellIndex, NeighbourUtils.Axis.Z))
```

Comment:
This was imported not statically, and it was good, leave it. If it takes too much space, add some enters.